code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -496,13 +496,14 @@
 		/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 6,
 		/obj/item/reagent_containers/hypospray/autoinjector/dylovene = 6,
 		/obj/item/reagent_containers/hypospray/autoinjector/hypervene = 4,
+		/obj/item/reagent_containers/hypospray/autoinjector/isotonic = 4,
 		/obj/item/reagent_containers/hypospray/autoinjector/quickclot = 4,
-		/obj/item/reagent_containers/hypospray/autoinjector/dexalinplus = 0,
+		/obj/item/reagent_containers/hypospray/autoinjector/peridaxon_plus = 0,
 		/obj/item/reagent_containers/hypospray/autoinjector/quickclotplus = 0,
 		/obj/item/reagent_containers/hypospray/autoinjector/synaptizine = 0,
-		/obj/item/storage/pill_bottle/bicaridine = 5,
-		/obj/item/storage/pill_bottle/kelotane = 5,
-		/obj/item/storage/pill_bottle/tramadol = 5,
```

Comment:
why

Reply:
per marinemed back down to three from five. Not EVERY unga needs to have pill bottles of these.

...unless their medics thi

Reply:
more justification needed beyond it shouldnt be

Reply:
I changed it because I wanted to.

You want me to change it back, gimme a reason to do so.